docs/sources/manage-users/_index.md

```diff
@@ -9,23 +9,54 @@ weight = 40
 
 # Manage users
 
-Create users and teams and configure [Permissions]({{< relref "../permissions/_index.md" >}}) to make sure that users only have access to the resources they need.
+Grafana offers several options for grouping users. Each level has different tools for managing user accounts and different tasks that they can perform.
 
-Only Administrators can manage users and teams.
+One of the most important user management tasks is assigning roles, which govern what [permissions]({{< relref "../permissions/_index.md" >}}) a user has. The correct permissions ensure that users only have access to the resources they need.
 
-## Users
+## Server
+
+The highest and broadest level of user group in Grafana is the server. Every user with an account in a Grafana instance is a member of the server group.
 
-Users are named accounts in Grafana that can be granted permissions to access resources throughout Grafana.
+Grafana Server Admins are user accounts that have the Grafana Admin option set to **Yes**. They can manage individual user accounts and organizations on their server.
 
+Server Admins can perform the following tasks:
+
+LINK TO MANAGE USERS IN SERVER
 - [Add or remove a user]({{< relref "./add-or-remove-user.md" >}})
 - [Enable or disable a user]({{< relref "./enable-or-disable-user.md" >}})
 
+## Organization
+
+Organizations are groups of users on a server. Users can belong to more than one organization, but each user must belong to at least one organization.
+
+Data sources, plugins, and dashboards are associated with organizations. This means that you could have a server with two organizations, one with a Prometheus data source and another with an InfluxDB data source. Each organization would have separate data and dashboards.
```

Comment:
I think this might be confusing. While one data source can be set up on one or more organizations, plugins and dashboards are tied to one organization. If the organization is deleted, then the dashboards are also deleted. Not sure about plugins.